config/serverless.yml

```diff
@@ -104,6 +104,9 @@ elasticsearch.ignoreVersionMismatch: true
 # Limit maxSockets to 800 as we do in ESS, which improves reliability under high loads.
 elasticsearch.maxSockets: 800
 
+# Hold the DNS lookup resolution for 60s max (if the DNS server claims a shorter TTL, it'll be used instead)
```

Comment:
nit: We need to say what changing the number of seconds improves, similarly to setting the es max sockets. 
We're exposing more config settings over time and the reason for exposing them should be documented.

Reply:
Done in [`cb3f3e0` (#184760)](https://github.com/elastic/kibana/pull/184760/commits/cb3f3e01e738a41e774c7241659093ae32f80dd9)